### PR TITLE
Pagerduty add config doc

### PR DESCRIPTION
**Pull request by BjMrq**
Add pager duty notification documentation channel form

### CODE REVIEW

**Review by shahnami — APPROVED** [review]
LGTM!

File: docs/modules/ROOT/pages/sentinel.adoc [hunk 1]
```diff
@@ -515,6 +515,21 @@ NOTE: It can take several minutes for a new custom metric to show up in the Data
...
 
+=== PagerDuty Configuration
+
+Please see https://support.pagerduty.com/docs/services-and-integrations[PagerDuty integration documentation] to configure an PagerDuty API v2 integration that can create change and alert events.
+
+* *Event Type* Event type for pager duty categorization (alert or change)
+* *Routing Key* Integration Key for an integration on a service or on a global ruleset (32 characters)
+* *Event Action* The action type of event (trigger, acknowledge or resolve)
+* *Dedup Key* Deduplication key for correlating triggers and resolves (255 max characters)
+* *Severity* The perceived severity of the status the event is describing with respect to the affected system  (critical, error, warning or info)
```
Comment (shahnami) [inline docs/modules/ROOT/pages/sentinel.adoc:526]: 2 whitespaces between `system` and the brackets

Comment (shahnami) [reply docs/modules/ROOT/pages/sentinel.adoc]: Hope you don't mind, just went ahead and removed it.

Comment (BjMrq) [reply docs/modules/ROOT/pages/sentinel.adoc]: Thank you very much! Did not see it on vs code will be more careful.
Thanks for the fix!

**Review by mok0230 — APPROVED** [review]

**Pull request merged**